### ISSUE TITLE
Resources added manually by lecturers are not detected

### ISSUE THREAD

**Issue by LloydTao** [LloydTao/ele-scraper#1]
In the expected state, resources that are added to the VLE are detected through the class `instancename`.

This appears to be a common convention across Moodle-based VLEs in other universities. 

However, some lecturers have added links to resources in a different fashion.

An example:
```
<table style="width: 716px; height: 736px;" cellspacing="0" cellpadding="0" border="1">
  <tbody>
    ...
    <tr>
      ...
      <td width="160" valign="top">
        <p>N-grams models <a href="https://vle.exeter.ac.uk/pluginfile.php/421246/course/section/87408/AI.17.N-grams.2020.pdf">(pdf)</a></p>
      </td>
      ...
    </tr>
    ...
  </tbody>
</table>
```

This will throw an exception such as:
```
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":".instancename"}
```

A potential fix may be to also look for links containing `pluginfile.php`. Note that these go directly to the resource, and don't follow the 'link -> resource' format of other files.

**Comment by LloydTao** [comment]
Working on this now.

**Comment by LloydTao** [comment]
Closed with the release of 1.2.0 :)
